news/2018-04-19-spring-2018-fanart-voting-now-open.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Spring 2018 Fanart Voting Now Open"
+date: 2018-04-19 14:00:00 +0000
+---
+
+Seventy-six amazing entries, and only ten days to judge them all! Talent is absolutely in bloom this year, with one of our most fiercely competitive fanart contests yet!
+
+[![](https://assets.ppy.sh/contests/64/header.jpg)](https://osu.ppy.sh/community/contests/64)
+
+With only **seven** votes to pass around, which will you choose from? Head on over to the [contest listing](https://osu.ppy.sh/community/contests/64) now and make your vote count - there's only **ten days** to get them in, so be quick about it!
+
+A selection of the top-voted entries will make it to the main menu as this year's Spring seasonal picks, which also includes an impromptu cameo during the finals of the osu!taiko World Cup! Talk about beating your drum, eh?
+
+The intrepid artists behind the winning works will nab themselves **2 months of osu!supporter** for their efforts - also giftable to a friend, two friends, or even four friends of their choice!
```

Comment:
2 months supporter to split between 4 people ? ._.

Reply:
the magic of assign supporter date

aka: anguish and suffering (but possible)

Reply:
your pain then, not mine :P